docs/source/programming_resources/shared/program_and_manage_network/Connecting-a-Laptop-to-the-Program-&-Manage-Network.rst

```diff
@@ -0,0 +1,131 @@
+Connecting a Laptop to the Program & Manage Network
+===================================================
+
+Connecting Your Laptop to the Program & Manage Network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to write an Op Mode, you will need to connect your programming
+laptop to the Program & Manage Wi-Fi network. The Program & Manage Wi-Fi
+network is a wireless network created by your Robot Controller. Before
+you begin this exercise, please make sure that your Windows laptop has
+the most current service pack and system update from Microsoft
+installed.
+
+Note that this example assumes the user has a Windows 10 laptop. If you
+are not using a Windows 10 laptop, the procedure to connect to the
+Programming & Manage Wi-Fi network will differ. Refer to your device's
+documentation for details on how to connect to a Wi-Fi network.
+
+Connecting Your Laptop to the Program & Manage Network Instructions
+-------------------------------------------------------------------
+
+1. On the Driver Station, touch the three dots in the upper right     
+hand corner of the screen to launch the pop-up menu. Select **Program 
+& Manage** from the pop-up menu to display the **Program & Manage**   
+access information.                                                   
+
+.. image:: images/SelectProgramAndManageDriverStation.jpg
+   :align: center
+
+|
+
+2. The Program & Manage screen displays important information that    
+you can use to connect your laptop to the FTC Blocks or OnBot Java    
+Programming Mode server.                                              
+
+.. image:: images/ProgramAndManageScreen.jpg
+   :align: center
+
+|
+
+3. Verify the network name and passphrase for the Program & Manage    
+wireless network. Towards the top of the screen, the name of the      
+Program & Manage wireless network is displayed. If you are using an   
+Android smartphone as your Robot Controller, then the wireless        
+network name will begin with the phrase "DIRECT-". 
+
+   In this example, the name of the Wi-Fi network is "DIRECT-XK-9999-C-RC" and the secure passphrase is "ZU7if0hB"                                              
+
+.. image:: images/ProgramAndManagePassphrase.jpg
+   :align: center
+
+|
+
+If you are using a Control Hub, then the wireless network name will be whatever you specified when you configured your Control Hub.  If you haven't changed the Control Hub's name yet, then by default the  wireless network's name will begin with "FTC-".  If you haven't changed its password yet, then by default the wireless network's passphrase will be "password".
+
+In the screenshot below, the Control Hub's wireless network name is "FTC-1Ybr" and the secure passphrase is "password".
```

Comment:
```
`FTC-1Ybr`
`password`
```

Reply:
I think this goes beyond the scope of the conversion.